editor/plugins/tiles/tile_data_editors.cpp

```diff
@@ -30,6 +30,9 @@
 
 #include "tile_data_editors.h"
 
+#include "scene/gui/menu_button.h"
+#include "scene/gui/option_button.h"
+#include "scene/gui/separator.h"
 #include "tile_set_editor.h"
```

Comment:
The intent here was likely to keep the "local" tileset includes first, before core/editor. scene includes should thus go further down (core + editor + scene could be grouped like we usually do).

Reply:
Seeing other `tiles` files, it seems @groud's preferred include style is:
```
// class header

// tile-related (local headers)

// editor

// scene

// core
```
That's not exactly the usual include style (https://docs.godotengine.org/en/latest/community/contributing/code_style_guidelines.html#header-includes) but if it's consistent within `tiles` it makes sense.